src/EventService.Data/EventRepository.cs

```diff
@@ -16,6 +16,18 @@ public EventRepository(IDataProvider provider)
     _provider = provider;
   }
 
+  public async Task<bool> CreateAsync(DbEvent dbEvent)
+  {
+    if (dbEvent == null)
+    {
+      return false;
+    }
+
+    _provider.Events.Add(dbEvent);
+    await _provider.SaveAsync();
+
+    return true;
+  }
   public Task<bool> DoesExistAsync(Guid eventId)
   {
     return _provider.Events.AsNoTracking().AnyAsync(e => e.Id == eventId);
```

Comment:
if the eventId exists but IsActive = false?

Reply:
Added check

Reply:
Where is IsActive?